src/cmdlinetest/test_afp_cli.t

```diff
@@ -48,6 +49,7 @@
    u?'--no-ask-pw': True, (re)
    u?'--show': False, (re)
    u?'--user': None, (re)
+   '--write': False,
```

Comment:
Why without the Unicode prefix? I am not sure if it matters, it just sticks out.